website/docs/r/dev_center.html.markdown

```diff
@@ -39,6 +39,8 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) Specifies the name of the Resource Group within which this Dev Center should exist. Changing this forces a new Dev Center to be created.
 
+* `dev_center_project_catalog_item_sync_enabled` - (Optional) Should project catalogs associated with projects in this Dev Center can be configured to sync catalog items?
```

Comment:
the resource name is dev_center so this should be
```suggestion
* `project_catalog_item_sync_enabled` - (Optional) Should project catalogs associated with projects in this Dev Center can be configured to sync catalog items?
```
?

Reply:
Updated